### ISSUE TITLE
Client Allocation Request for: hanatourist

### ISSUE THREAD

**Issue by latoyahdail** [filecoin-project/filecoin-plus-client-onboarding#749]
---

# Client Allocation Questions 

## Core Information
- Name: hanatourist
- Website / Social Media: https://hanatourist.vip
- Region: Asia excl. Greater China
- DataCap Requested: 5TiB
- Addresses to be Notarized: f1e4ie6hm6zkdro24ykupqvnt6m6jt5kd2ab4fz6i
- Notary Requested: rayshitou

@latoyahdail Please subscribe to notifications for this Issue to be aware of updates. Notaries may request additional information on the Issue.

**Comment by rayshitou** [comment]
@latoyahdail Thanks for your application. Please answer the following questions to let me know you and your company.

1.What's your name and your position in your company?What's your company's business?
2.Could you show me your company's business license? And please show me your company searching consequence in the local business license searching website.
3.Have you/your company ever used cloud? Which platform do you use? What's your usage volume and frequency?Could you please show me your cloud usage evidence?

**Comment by latoyahdail** [comment]
@rayshitou   Sorry to reply so late.

1. You can call me Randy, I am the technical director at Hanatourist. 
Hanatourist is a well-known local tourism integrated service company in Vietnam. We have been established for more than 20 years. We are committed to providing the best service for tourists who come to Vietnam for tourism. Due to the diversified business development, it began to get involved in the digital information industry. My department is mainly responsible for researching the latest digital technology.

2. Sure but for privacy I will send you via email.

3. We usually use VDC's cloud service, which is about 5T per month. About the evidence I can not provide for privacy.

**Comment by EVE1011** [comment]
Dear@Broz221  Glad to see your reply

1.We think more than 3 is reasonable,It mainly depends on the scale of the miner and the price.

2.The price of their services geographical location and technical capabilities.

3.We will allocate DataCap reasonably to prevent overallocation.

**Comment by rayshitou** [comment]
Please send your company's business license and searching consequence to dlleig@gmail.com.
And please let me know:
1.What kind of data do you want to store and how much?
2.Could you please send me some demos of your data?
3.Do you agree to use the DataCap to only store data that abides by local regulations and in compliance with the recipient miner's terms of service?

**Comment by latoyahdail** [comment]
@rayshitou Sorry to reply so late.

1.I want to store the travel data of our users, including photos and videos. Around 5T per month.

2.Because it is customer privacy, I can find some data approved by the customer and send it to your email.

3.yes.

**Comment by rayshitou** [comment]
I'm a little confused. Private data is not recommended to storing in Filecoin system.

**Comment by latoyahdail** [comment]
Yes, we checked the rules of Filecoin Plus again and indeed we can't store private data. So we decided to store data about our own company generated data. Mainly videos of local people and scenery and photos.

**Comment by rayshitou** [comment]
About the allocation of DataCap:
1.How many miners do you want to store your data to? How much data for each miner?
2.What method/tool will you use to insperct miners? Which aspect of miners will you inspect?
3.Will you record your allocation decision in public? Which platform will you use?
4.Have you ever allocated before? If yes, how did you do when you were questioned? Any links?
